mteb/tasks/Reranking/multilingual/MIRACLReranking.py

```diff
@@ -4,35 +4,88 @@
+    issn = {2307-387X},
+    doi = {10.1162/tacl_a_00595},
+    url = {https://doi.org/10.1162/tacl\_a\_00595},
+    eprint = {https://direct.mit.edu/tacl/article-pdf/doi/10.1162/tacl\_a\_00595/2157340/tacl\_a\_00595.pdf},
+}"""
 
 
 class MIRACLReranking(MultilingualTask, AbsTaskReranking):
     metadata = TaskMetadata(
         name="MIRACLReranking",
-        description="MIRACL (Multilingual Information Retrieval Across a Continuum of Languages) is a multilingual retrieval dataset that focuses on search across 18 different languages. This task focuses on the German and Spanish subset.",
+        description="MIRACL (Multilingual Information Retrieval Across a Continuum of Languages) is a multilingual retrieval dataset that focuses on search across 18 different languages.",
         reference="https://project-miracl.github.io/",
         dataset={
-            "path": "jinaai/miracl",
-            "revision": "d28a029f35c4ff7f616df47b0edf54e6882395e6",
+            "path": "miracl/mmteb-miracl-reranking",
+            "revision": "6d1962c527217f8927fca80f890f14f36b2802af",
         },
         type="Reranking",
         category="s2s",
-        eval_splits=["test"],
-        eval_langs={
-            "de": ["deu-Latn"],
-            "es": ["spa-Latn"],
-        },
-        main_score="map",
-        date=None,
-        form=None,
-        domains=None,
-        task_subtypes=None,
-        license=None,
-        socioeconomic_status=None,
-        annotations_creators=None,
-        dialect=None,
-        text_creation=None,
-        bibtex_citation=None,
+        eval_splits=[_EVAL_SPLIT],
+        eval_langs=_LANGUAGES,
+        main_score="NDCG@10",
+        date=("2022-06-01", "2023-01-30"),
+        form=["written"],
+        domains=["Encyclopaedic"],
+        task_subtypes=[],
+        license="CC BY-SA 4.0",
+        socioeconomic_status="low",
+        annotations_creators="expert-annotated",
+        dialect=[],
+        text_creation="created",
+        bibtex_citation=_CITATION,
         n_samples=None,
         avg_character_length=None,
```

Comment:
Let is add these two as well

Reply:
ah I thought about to ask but forgot. What's the definition for `n_samples` and `avg_character_length` here? Does `n_samples` mean the number of queries or all query-document pairs? and is `avg_character_length` the average char length of all the queries and documents?

Reply:
You can calculate it using the utility function `task.calculate_length_and_count()` (the formulation isn't perfect but it is consistent)

n_samples is the number of pairs and total length is the sum of length of query and doc in those pairs.

Reply:
sg thanks! I'll add them

Reply:
added!